src/build_test.cc

```diff
@@ -1750,15 +1800,17 @@ TEST_F(BuildTest, RspFileSuccess)
   EXPECT_TRUE(builder_.Build(&err));
   ASSERT_EQ(3u, command_runner_.commands_ran_.size());
 
-  // The RSP files were created
-  ASSERT_EQ(files_created + 2, fs_.files_created_.size());
+  // The RSP files and temp file to acquire output mtimes were created
+  ASSERT_EQ(files_created + 3, fs_.files_created_.size());
   ASSERT_EQ(1u, fs_.files_created_.count("out 2.rsp"));
   ASSERT_EQ(1u, fs_.files_created_.count("out 3.rsp"));
+  ASSERT_EQ(1u, fs_.files_created_.count(".ninja_tmp"));
```

Comment:
IMHO this is an implementation detail and shouldn't be part of the test.

Reply:
the file is created by the virtual file system during the test; putting the assert here makes it clear why there were 3 files created.

Reply:
Still, I would rather remove the explicit check for `ASSERT_EQ(files_created + 2, fs_.files_created_.size());`. *This* unit test shouldn't care.

Reply:
Ok, I'll remove it.